.devcontainer/localstack/init.sh

```
@@ -1,39 +1,39 @@
-            "AllowedOrigins": [
-                "*"
-            ],
-            "ExposeHeaders": []
-        }
-    ]
-}'
-
-awslocal s3 mb s3://$bucketName
-
-awslocal s3api put-bucket-cors --bucket $bucketName --cors-configuration "${corsConfig}"
+#!/bin/bash
```

Shellcheck (0.10.0)

[error] Literal carriage return. Run script through tr -d '\r' .

(SC1017)